src/CounterSegment.js

```diff
@@ -137,13 +110,9 @@ class CounterSegment extends React.Component {
    * @return {ReactElement} counter segment
    */
   render () {
-    const style = {
-      overflow: 'hidden',
-      height: `${this.state.digitHeight}px`
-    }
     return (
       <div className='rollex-segment'>
-        <div className='rollex-digits' style={style}>
+        <div className='rollex-digits' style={{ overflow: 'hidden' }}>
```

Comment:
`overflow: hidden` does not have the same problem as `height` does. Overflow is unlikely to be overridden (only if someone wants to literally break the counter), while `height` pretty much has to be overridden.